<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StegAWOO</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBoxStoringMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1.1 Seq Frame Seq Pixel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.2 Seq Frame Rand Pixel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.1 Rand Frame Seq Pixel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.2 Rand Frame Rand Pixel</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="storingLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Storing Mode</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioModeStore">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Store</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">modeButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioModeRetrieve">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Retrieve</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">modeButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="modeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="encryptionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Encryption</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioEncryptionWithout">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Without</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">encryptionButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioEncryptionWith">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>With (Vigenere)</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">encryptionButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="lineEditInputVideoLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditInputVideo">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonInputVideo">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonFileInput">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditInputFile">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lineEditInputFileLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File to Hide</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonOutputVideo">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOutputVideo">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lineEditOutputVideoLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output  File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonProcess">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>150</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QLabel" name="keyLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditKey">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>140</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Key will be ignored if without encryption</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonGetMetadata">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>190</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Metadata</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Metadata</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame Count</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filename</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ext</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFrameCount">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFilename">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelExt">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPSNR">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate PSNR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PSNR</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPSNR">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonOpenInput">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Input</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonOpenFile">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File to Hide</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonOpenOutput">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Output</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="modeButtonGroup"/>
  <buttongroup name="encryptionButtonGroup"/>
 </buttongroups>
</ui>
